DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi/FrontEnd/src/client/app/activity/activity.directive.js

```diff
@@ -0,0 +1,42 @@
+(function () {
+    'use strict';
+
+    angular.module('app.activity')
+        .directive('geActivity', activity);
+
+    activity.$inject = [
+        '$templateCache',
+        '$compile',
+        'logger',
+        'ACTIVITY_CONSTANTS',
+        'i18n'
+    ];
+
+    function activity($templateCache, $compile, logger, constants, i18n) {
+        function link(scope, element) {
+            var type = scope.activity.type;
+
+            if (!type) {
+                logger.error(constants.UNKNOWN_TYPE_MESSAGE);
+                return;
+            }
+
+            var template = $templateCache.get(
+                constants.ACTIVITY_PATH_PREFIX + type + constants.ACTIVITY_PATH_POSTFIX
+            );
+            element.html(template).show();
+            scope.i18n = i18n.message;
+
```

Comment:
```js
var template = $templateCache.get(
constants.ACTIVITY_PATH_PREFIX + type + constants.ACTIVITY_PATH_POSTFIX
);
element.html(template).show();
$compile(element.contents())(scope);
```
is a logical block of code. please don't mix it with `scope.i18n = i18n.message;` just move `scope.i18n = i18n.message;` on top